### PR TITLE
Release 8.0.0

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
...
+- Support alternative package & service name for Kibana [\#16](https://github.com/voxpupuli/puppet-kibana/issues/16)
+- Add Puppet 8 support [\#76](https://github.com/voxpupuli/puppet-kibana/pull/76) ([bastelfreak](https://github.com/bastelfreak))
+- Remove template erb and puppet\_x folder [\#72](https://github.com/voxpupuli/puppet-kibana/pull/72) ([phaedriel](https://github.com/phaedriel))
+- Add optional parameter plugindir \(Directory containing kibana plugins\) [\#71](https://github.com/voxpupuli/puppet-kibana/pull/71) ([phaedriel](https://github.com/phaedriel))
+- Add sensitive for kibana config [\#68](https://github.com/voxpupuli/puppet-kibana/pull/68) ([phaedriel](https://github.com/phaedriel))
+
+**Closed issues:**
+
+- kibana plugin installation and directory permissions issue [\#69](https://github.com/voxpupuli/puppet-kibana/issues/69)
+- bundle exec rake spec failed [\#62](https://github.com/voxpupuli/puppet-kibana/issues/62)
```
Comment (bastelfreak) [inline CHANGELOG.md:26]: I added the `skip-changelog` label to both. please regenerate the changelog, then both issues should disappear.

**Review by smortex — COMMENTED** [review]
I fixed some labels, can you regenerate the ChangeLog?

**Comment by rwaffen** [comment]
can do... one mom

File: CHANGELOG.md (hunk 1 above)
Comment (smortex) [reply CHANGELOG.md]: Ah, I removed "enhancement" from #16 so now it appears here.  Do you want to skip-changelog on it too?

Comment (bastelfreak) [reply CHANGELOG.md]: I think it's fine now.

Comment (rwaffen) [reply CHANGELOG.md]: okay, if we are fine here (?) i'm going to merge it

**Pull request merged**